app/frontend/components/pages/exportWallet/exportWalletPage.js

```diff
@@ -155,7 +155,15 @@ class ExportWalletDialog extends Component {
             }),
             h(
               'div',
-              {class: 'validation-row'},
+              {class: 'export-content-bottom'},
+              h(
+                'button',
+                {
+                  class: 'button secondary',
+                  onClick: () => window.history.back(),
```

Comment:
when you enter the https://<host:port>/exportWallet page (directly, not via the login page), it first redirects you to the login page, and when you log in, it redirects you to the export wallet page, but the back button does not work as expected (you need to click it several times until you get to the wallet page).

I would suggest fixing this as well - my idea is that if you login via adalite it should bring you always to the txHistory page to avoid the conflict of what the "back" button is supposed to do if you did not really come from the txHistory page